package.json

```diff
@@ -156,7 +156,8 @@
         "command": "one.toolchain.install",
         "title": "Toolchain Install",
         "category": "ONE",
-        "icon": "$(add)"
+        "icon": "$(add)",
+        "enablement": "!one.Job:running"
```

Comment:
(optional) Can you change `one.Jon:running` into `one.job:running`? When I changed the command names, I used camelcases.

Reply:
Okay. I will update PR.